README.md

```diff
@@ -1 +1,97 @@
-# Covid-19-app
\ No newline at end of file
+# CovidTracker--SARS-CoV-2.
+
+> This webapp provides Covid-19 Tracking information using the data it gets from an [API data source](https://corona.lmao.ninja) to provide comprehensive information about the novel coronavirus,
+>
+> It is created with [Create-React-App](https://github.com/facebook/create-react-app)
+
+---
+## Video Presentation
+(https://www.loom.com/share/8f32d2752b184da9ae05a6810dd2fd40)
+
+## Live version
+
+For the live version of this project visit the followinglink.
+ [Netlify](https://verdant-griffin-3590dd.netlify.app)
+
+
+## Technology
+
+- React-Create-App
+- React-DOM
+- Redux
+- npm
+- Javascript ES6
+- Nodejs
+- HTML && CSS
+
+## Getting Started
+
+**This are steps on how to install the react app.**
+
+To get a local copy up and running follow these simple example steps.
+
+### `clone repository on your local device`
+
+copy this link to get HTTP url to clone Into your device `https://github.com/Fatima-hub333/React-Capstone.git`
+
```

Comment:
Please update your Getting started section and make sure that the url for cloning your project is for your repository.